packages/playground/src/dashboard/components/create_farm.vue

```diff
@@ -71,18 +71,20 @@ export default {
       required: true,
     },
   },
-  setup(props) {
+
+  setup(props, context) {
     const showDialogue = ref(false);
     const isCreating = ref(false);
     const gridStore = useGrid();
     const valid = ref(false);
+
     async function createFarm() {
       try {
         isCreating.value = true;
         await gridStore.grid.farms.create({ name: props.name });
         createCustomToast("Farm created successfully.", ToastType.success);
-        notifyDelaying();
```

Comment:
why do you remove it? i think it will be better to notify the user that it will reload in a while

Reply:
added back